### PR TITLE
Feature - Implemented pulling crates.io packages

### PR DESCRIPTION
**Pull request by naveensrinivasan**
Closes #23

### CODE REVIEW

**Comment by jordan-wright** [comment]
@naveensrinivasan just a heads up, we'll want to remove the `crates` binary from the PR before merging.

**Comment by naveensrinivasan** [comment]
> @naveensrinivasan just a heads up, we'll want to remove the `crates` binary from the PR before merging.

@jordan-wright Thanks. 🤦‍♂️

**Comment by naveensrinivasan** [comment]
> > @naveensrinivasan just a heads up, we'll want to remove the `crates` binary from the PR before merging.
> 
> @jordan-wright Thanks. 🤦‍♂️

Removed the `crates` binary

File: feeds/crates/main.go [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+	}
+
+	//nolint:errcheck
+	log.Printf("Processed %d packages", processed)
+	//nolint:errcheck
+	w.Write([]byte("OK"))
+}
+
+func main() {
+	log.Print("polling npm for packages")
```
Comment (dlorenc) [inline feeds/crates/main.go:121]:
```suggestion
	log.Print("polling crates for packages")
```

**Review by dlorenc — APPROVED** [review]
One little typo

File: feeds/crates/main.go (hunk 1 above)
Comment (naveensrinivasan) [reply feeds/crates/main.go]: Thanks 👍

**Pull request merged**